src/game.cc

```diff
@@ -209,16 +212,16 @@ void Game::UpdateHighScore()
 {
   if (curr_score > curr_high_score)
     difficulty_map->operator[](curr_difficulty) = curr_score;
-  
+
 
   *prev_score = curr_score;
 
   wclear(info_win);
   mvwprintw(info_win, 7, 0, "high score: %d", curr_high_score);
   mvwprintw(info_win, 8, 0, "difficulty_map difficulty: %d", difficulty_map->at(curr_difficulty));
-  
+
   wrefresh(info_win);
-  
+
```

Comment:
nice job getting rid of the whitespaces

Reply:
This was my VScode [Trailing Spaces plugin](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) just automatically doing it for me. Was too lazy to turn it off just for this change lol. I wish I had this plugin when we were first developing this.